src/components/CommentList/CommentList.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Comment } from '../Comment/Comment';
+import { ProtoPost } from '../ProtoPost';
+
+export const CommentList = ({ post }) => (
+  <div>
+    {post.commentList.map(comment => (
+      <div className="comment" key={post.id}>
+        <Comment comment={comment} />
```

Comment:
If you use spread operator here, you'll be able to easily destructure needed properties inside `Comment` component.
```suggestion
        <Comment comment={...comment} />
```